README.md

```diff
@@ -151,15 +159,15 @@ Local development requires:
 
 `./src/functions/github.js`
 
-The aim of this lammbda is to have up to date metrics of yld's open source contribution on the site. We do this by aggregating data on a lambda and publishing it to Contentful. It is split into two sections, `repos` and `meta`.
+The aim of this lambda is to have up to date metrics of yld's open source contribution on the site. We do this by aggregating data on a lambda and publishing it to Contentful. It is split into two sections, `repos` and `meta`.
 
 - Repos
-  We want to store data regarding specific repos that members of yld have contributed to e.g. node, react, enyzyme etc. By creating a `githubRepo` content type on Contentful with only a URL value edtiable, we are able to create references to these repos that can be used throughout the site but have metrics that are _only_ available to update via the API medium. This ensures data is always valid and accurate.
+  We want to store data regarding specific repos that members of yld have contributed to e.g. node, react, enzyme etc. By creating a `githubRepo` content type on Contentful with only a URL value editable, we are able to create references to these repos that can be used throughout the site but have metrics that are _only_ available to update via the API medium. This ensures data is always valid and accurate.
 
 - Meta
   Meta data is used a summary of all the repositories that yldio organisation has contributed to. This data is written directly to the Open Source content type.
 
-Main [`@yld.io/oss-stats`](https://www.npmjs.com/package/@yldio/oss-stats) to aggregate all open source contributions for `yldio` organization members.
+Main [`@yld.io/oss-stats`](https://www.npmjs.com/package/@yldio/oss-stats) to aggregate all open source contributions for `yldio` organisation members.
```

Comment:
Since other places were using `organisation` (British), it's better to standardize.